### ISSUE TITLE
Installer isn't installing all files

### ISSUE THREAD

**Issue by yoGhurrt1** [hadouken/hadouken#154]
I had to use .zip package because when i figured it out, msi package installer only hadouken.exe in default directory.
Edit1 :Obviously, wizard isn't configuring hadouken.json config file neither.
Edit2 :Ok, it's configured but it's stored in C:\ProgramData\Hadouken - but .exe is looking for it in directory of hadouken, not in "Program Data"
Edit3 : I probably should create new issue for all of this, anyway, configured file via wizard add's wrong paths to "statePath" and "scripting". In .json file, there are too many and wrong slashes like i.e "C:/ProgramData/Hadouken//state/"
Edit4 : But when i'll change slashes to right one, i've got error "Invalid escape sequence"

**Comment by vktr** [comment]
On a standard Windows installation (with Windows installed on the C drive) Hadouken will put `hadouken.exe` in `C:\Program Files (x86)\Hadouken` and then read the configuration file from `C:\ProgramData\Hadouken\hadouken.json`. It will also have the JS files in the `C:\ProgramData\Hadouken\js` directory.

Could you post an example of the `hadouken.json` file after installing, showing the incorrect configuration?

**Comment by yoGhurrt1** [comment]
Yes, it should read configuration from there, but when I'll run hadouken.exe from console, error says that hadouken.exe cannot find config file named hadouken.json in his directory, so, C:\Program Files (x86)

\Hadouken\ , not in ProgramData directory.

**Comment by yoGhurrt1** [comment]
Here, i'm pasting whole config file that i made for testing:

{
    "bittorrent":
    {
        "defaultSavePath": "D:/Test",
        "statePath": "C:/ProgramData/Hadouken//state/",

        "anonymousMode": false,

        "dht":
        {
            "enabled": true,
            "routers":
            [
                [ "router.bittorrent.com",  6881 ],
                [ "router.utorrent.com",    6881 ],
                [ "dht.transmissionbt.com", 6881 ],
                [ "dht.aelitis.com",        6881 ]
            ]
        },

        "proxy":
        {
            "type": "none",
            "hostname": "",
            "port": 0,
            "username": "",
            "password": "",
            "proxyHostnames": true,
            "proxyPeerConnections": true
        },

        "storage":
        {
            "sparse": true
        },

        "upnp":
        {
            "enabled": true
        }
    },

    "extensions":
    {
        "autoadd":
        {
            "enabled": false
        },

        "automove":
        {
            "enabled": false
        },

        "launcher":
        {
            "enabled": false
        },

        "pushbullet":
        {
            "enabled": false,
            "token": "YOUR-PUSHBULLET-AUTH-TOKEN",
            "enabledEvents":
            [
                "hadouken.loaded",
                "torrent.added",
                "torrent.finished"
            ]
        }
    },

    "http":
    {
        "address": "0.0.0.0",
        "port": 7070,

        "auth":
        {
            "type": "basic",
            "token": "",
            "basic":
            {
                "userName": "Test",
                "password": "test"
            }
        },

        "root": "/",

        "ssl":
        {
            "enabled": false,
            "privateKeyFile": "",
            "privateKeyPassword": ""
        }
    },

    "scripting":
    {
        "path": "C:/ProgramData/Hadouken//js"
    }
}

**Comment by vktr** [comment]
The problem is that when running Hadouken as a Windows Service (which it does by default when using the installer) its working directory is somewhere in the `Windows` folder. Not good, since we can't write anything there. So we read configuration from the `ProgramData` which is what MSDN recommends.

However, when running as a console application, we assume we have read/write rights to the current working directory and we try to read the configuration file from there.

So, in short,
- Windows Service: `C:\ProgramData\Hadouken\hadouken.json`
- Console Application: `%CWD%\hadouken.json`

**Comment by yoGhurrt1** [comment]
Ok, i get it now how hadouken works. But, now i'm not sure if log in ProgramData will be enough to get some detailed information, to write in an issue if there would be a new problem. 
Also still, i'm not sure if those slashes should look that way, i'll try to configure hadouken again today.

**Comment by vktr** [comment]
If you're putting back slashes (`\`) in the JSON file, use double back slashes, like `C:\\My\\Path`. Otherwise, forward slashed works also.
